src/Frequency/buffer.jl

```diff
@@ -1,4 +1,29 @@
-abstract type Buffer end
+"""
+    Buffer
+
+Struct used for reading out vertices from Action struct. 
+Contains symmetry related flags, (asymptotic) kernel specification and interpolation parameters.
+* `exchange_flag :: Bool`  : flag for site [(i0, j) <-> (j, i0)] and spin [(μ, ν) <-> (ν, μ)] exchange
+* `map_flag      :: Bool`  : flag for channel mapping s <-> u
+* `sgn_μν        :: Bool`  : sign function for combined spin indices 
+* `sgn_μ         :: Bool`  : sign function for first spin index 
+* `sgn_ν         :: Bool`  : sign function for second spin index 
+* `kernel        :: Int64` : specification of asymptotic kernel to be interpolated
+* `p1            :: Param` : interpolation parameters for bosonic frequency argument
+* `p2            :: Param` : interpolation parameters for first fermionic frequency argument
+* `p3            :: Param` : interpolation parameters for second fermionic frequency argument
+"""
+struct Buffer
+    exchange_flag :: Bool
+    map_flag      :: Bool 
+    sgn_μν        :: Bool
```

Comment:
Is the combined sign really necessary, if we have the single ones?

Reply:
I initially thought yes, there are symmetries which give directly the product xi_mu * xi_nu, whereas other only give you a factor xi_mu or xi_nu. However, I see your point, probably combined symmetries which e.g. give a combination xi_mu^2 * xi_nu can also be captured if only the single factors are saved, but I might have had a reason to it like this (been a while).

Reply:
Let's leave it with how it is for now, will raise an issue to look into that further.